frontend/amundsen_application/api/utils/metadata_utils.py

```diff
@@ -128,6 +134,8 @@ def marshall_table_full(table_dict: Dict) -> Dict:
     for col in columns:
```

Comment:
I feel like it is a little weird to have this functionality where we iterate through columns and set editable state in the marshal function. Maybe we have a function that takes columns and maps them to whatever schema we expect similar to for other properties here like owners? That way if there is any further metadata transformation needed for columns it can happen there instead?

Reply:
I looked into it a bit more, and the `is_editable` field is not a part of the table or column schemas, and according to the comment for this marshall function it is meant to take care of "additional or sanitized fields". So this may not be the "best" place to do it but it does seem to fit in with the purpose of this function for now, and it is a little different than the owner situation since that one is filling in fields that are in the schema but just aren't populated by the metadata service

Reply:
is_editable should almost certainly be on the schema so it can be a normal field and not this snowflake thing. But maybe this is not the time to go and refactor that. I vote for leaving a TODO.